### PR TITLE
Community resources page

### PR DESCRIPTION
**Pull request by LoisRP**
## High Level Overview of Change

Changed Community Tools to Community Resources and added more links, as described here:
https://app.asana.com/0/1138337311010009/1182472031447793/f

### Context of Change

We want the docs to highlight community contributions. 

### Type of Change

- [x ] Documentation Updates

## Before / After

Current page points to Validator only.
New page points to various tools. 

## Test Plan

Built and tested locally. 

## Future Tasks
Add new resources and tools as they become available.

### CODE REVIEW

File: docs/community-resources.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+id: community-resources
+title: Community Resources
+sidebar_label: Community Resources
+---
+
+Members of the PayID community have created useful resources for using and learning about PayID.
+
+- [PayID Youtube Tutorial by Ken Carabello](https://www.youtube.com/watch?v=gpLJp1lylns)
```
Comment (hbergren) [inline docs/community-resources.md:9]: Can we add some subheaders?

Maybe:
- Tutorials
- PayID Server Implementations
- Tools
- Community (Things like the subreddit, and Discord channel).

File: docs/community-resources.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  - [HTTP response headers](payid-headers)
+  - [Cache-Control](payid-best-practices#cache-control)
+  - [CORS accessibility](payid-best-practices#set-cors-cross-origin-resource-sharing-headers)
+  - Response time
+
+  See also: [Python-based PayID validator](https://github.com/RockHoward/python-payid-validator).
+
+- [PayID Server](https://github.com/RockHoward/django-payid-server) - A Django-based PayId reference server.
+
+- [Assorted PayID Client Libraries in Node.js](https://www.npmjs.com/search?q=payid)
```
Comment (hbergren) [inline docs/community-resources.md:22]: I wouldn't link the NPM search query directly.

File: docs/community-resources.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+- The [PayID Validator](https://payidvalidator.com/) is an [open-source project](https://github.com/rswarthout/payid-validator) that validates responses using [JSON schemas](https://docs.payid.org/payid-interfaces) and verifies whether your PayID server follows various [best practices](payid-best-practices), like:
+
+  - [HTTP response headers](payid-headers)
+  - [Cache-Control](payid-best-practices#cache-control)
+  - [CORS accessibility](payid-best-practices#set-cors-cross-origin-resource-sharing-headers)
+  - Response time
+
+  See also: [Python-based PayID validator](https://github.com/RockHoward/python-payid-validator).
+
+- [PayID Server](https://github.com/RockHoward/django-payid-server) - A Django-based PayId reference server.
```
Comment (hbergren) [inline docs/community-resources.md:20]:
```suggestion
- [Django PayID Server](https://github.com/RockHoward/django-payid-server) - A Django-based PayID server.
```

File: docs/community-resources.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+- [Assorted PayID Client Libraries in Node.js](https://www.npmjs.com/search?q=payid)
+
+- [PayID Static](https://github.com/WietseWind/PayID-Static)
```
Comment (hbergren) [inline docs/community-resources.md:24]: This is a 
```suggestion
- [PayID-Static](https://github.com/WietseWind/PayID-Static) - A Node.js PayID server with static configuration (easy to use on Heroku).
```

File: docs/community-resources.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+- [PayID Youtube Tutorial by Ken Carabello](https://www.youtube.com/watch?v=gpLJp1lylns)
+
+- The [PayID Validator](https://payidvalidator.com/) is an [open-source project](https://github.com/rswarthout/payid-validator) that validates responses using [JSON schemas](https://docs.payid.org/payid-interfaces) and verifies whether your PayID server follows various [best practices](payid-best-practices), like:
+
+  - [HTTP response headers](payid-headers)
+  - [Cache-Control](payid-best-practices#cache-control)
+  - [CORS accessibility](payid-best-practices#set-cors-cross-origin-resource-sharing-headers)
+  - Response time
+
+  See also: [Python-based PayID validator](https://github.com/RockHoward/python-payid-validator).
```
Comment (hbergren) [inline docs/community-resources.md:18]: This isn't really a "See also", these tools validate different things.

The PayID Validator validates PayID server responses.

The "Python-based PayID validator" validates URI strings and sees if they are valid PayIDs.

File: docs/community-resources.md (hunk 1 above)
Comment (LoisRP) [reply docs/community-resources.md:9]: Will do.

File: docs/community-resources.md (hunk 2 above)
Comment (LoisRP) [reply docs/community-resources.md:22]: Will change the wording. How about: 
Search for PayID Client Libraries in [npm](https://www.npmjs.com)

Comment (hbergren) [reply docs/community-resources.md:22]: I don't think it's useful to link to NPM at all. I think we should have a curated list of resources that are reasonably high-quality, which just searching NPM won't give us.

Comment (LoisRP) [reply docs/community-resources.md]: Removed

File: docs/community-resources.md [hunk 6]
```diff
@@ -7,22 +7,26 @@ sidebar_label: Community Resources
...
 
   - [HTTP response headers](payid-headers)
   - [Cache-Control](payid-best-practices#cache-control)
   - [CORS accessibility](payid-best-practices#set-cors-cross-origin-resource-sharing-headers)
   - Response time
 
-* [Python-based PayID validator](https://github.com/RockHoward/python-payid-validator) - Validates URI strings to determine if they are valid PayIDs.
+- [Python-based PayID validator](https://github.com/RockHoward/python-payid-validator) - Validates URI strings to determine if they are valid PayIDs.
 
 **Reference server implementations**
```
Comment (hbergren) [inline docs/community-resources.md:24]:
```suggestion
**PayID server implementations**
```

I don't think we should call these "reference servers". Only our server at `payid-org/payid` is the reference server. These are just other PayID server implementations.

**Pull request merged**